docs/docsite/rst/community/reporting_bugs_and_features.rst

```diff
@@ -15,36 +16,44 @@ Reporting a bug
 Security bugs
 -------------
 
-Ansible practices responsible disclosure - if this is a security-related bug, email `security@ansible.com <mailto:security@ansible.com>`_ instead of filing a ticket or posting to any public groups, and you will receive a prompt response.
+Ansible practices responsible disclosure - if this is a security-related bug, email `security@ansible.com <mailto:security@ansible.com>`_ instead of filing a ticket or posting to any public groups to receive a prompt response.
 
 Bugs in ansible-core
 --------------------
 
-If you find a bug that affects multiple plugins, a plugin that remained in the ansible/ansible repo, or the overall functioning of Ansible, report it to `github.com/ansible/ansible/issues <https://github.com/ansible/ansible/issues>`_. You need a free GitHub account.  Before reporting a bug, use the bug/issue search to see if the issue has already been reported. If you are not sure if something is a bug yet, you can report the behavior on the :ref:`mailing list or community chat first <communication>`.
+Before reporting a bug, use the bug/issue search to check for :ref:`already reported issues <https://github.com/ansible/ansible/issues>`. Unsure if you found a bug? Report the behavior on the :ref:`mailing list or community chat first <communication>`.
+
+Also, use the mailing list or chat if you are unsure whether a bug is in ``ansible-core`` or in a collection, and for "how do I do this" type questions to discuss.
+
+You need a free GitHub account to `report bugs <https://github.com/ansible/ansible/issues>`_ that affects
```

Comment:
Idk how to write the link syntax to make the CI happy :/

Reply:
[`github.com/ansible/ansible/issues <https://github.com/ansible/ansible/issues>`_](https://sublime-and-sphinx-guide.readthedocs.io/en/latest/references.html#links-to-external-web-pages)

> `Link text <link URL>`_

This is what I have done.

Reply:
It seems like CI is happy with this one now and is failing on the one above:
```
:ref:`already reported issues <https://github.com/ansible/ansible/issues>`.
```

Reply:
Hi @cadamini and thanks for taking up one of the Docs issues and fixing with this PR!  So the links are definitely tricky and I have to look them up all the time myself.
Generally if it is a link based on an RST anchor, it is :ref:`foo_anchor`.   If it is an external url, it is `foo <foo.com>`_.  So for the remaining problem, it would be `already reported issues <https://github.com/ansible/ansible/issues>`_.